serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -119,4 +123,4 @@ LABEL djl-version=$djl_version
 LABEL trtllm-version=$trtllm_version
 LABEL cuda-version=$cuda_version
 # To use the 535 CUDA driver
-LABEL com.amazonaws.sagemaker.inference.cuda.verified_versions=12.2
+LABEL com.amazonaws.sagemaker.inference.cuda.verified_versions=12.4
```

Comment:
we don't have 12.4 label yet, we need to check with Ace on this

Reply:
Changed back to 12.2